### DISCUSSION TITLE
Smarter ways to sync to remote Cryptomator vaults

### DISCUSSION THREAD

**Discussion by brunescio** [iterate-ch/cyberduck#16177] [Q&A]
I have some vaults on an FTP server that I have created using Cyberduck.

Problem is, the synchronize feature of Cyberduck only allows Upload, Download and Mirror.

I'd rather have something like `rsync -uva --delete`.

Does anybody have some ideas?

**Comment by dkocher** [comment]
I suppose you are referring in particular to the `--delete` (_delete extraneous files from dest dirs_) option. We track this feature request in [^2].

[^2]:  #12171

**Reply by brunescio** [reply]
Exactly that! Do you know if this will be implemented anytime soon?

Also, do you have an idea for a workaround in the meantime? 🙂

**Comment by boboG** [comment]
I'd like to stress that a better way of deleting files in Cryptomator vaults is needed. I'm currently trying to delete a few ten thousand files from a vault located on a OneDrive. Mountain Duck is deleting the files one by one ... counting down like 2 / second.
